### PR TITLE
fixing url in SameSite cookie recipes blog

### PR DESCRIPTION
**Pull request by digi-booster**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

- 
- 
-

### CODE REVIEW

File: src/site/content/en/blog/samesite-cookie-recipes/index.md [hunk 1]
```diff
@@ -284,6 +284,6 @@ anyone has encountered it - so don't hesitate to reach out:
   [`SameSite` updates page](https://www.chromium.org/updates/same-site).
 
 _Cookie hero image by
-[Cayla1](https://unsplash.com/@calya1?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText)
+[Calajay](https://unsplash.com/@calajay)
 on
 [Unsplash](https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText)_
```
Comment (samthor) [inline src/site/content/en/blog/samesite-cookie-recipes/index.md:289]: Remove the UTM stuff here too.

Comment (digi-booster) [reply src/site/content/en/blog/samesite-cookie-recipes/index.md:289]: How can I find the same link?

**Comment by samthor** [comment]
@digi-booster remove the UTM content please

**Comment by digi-booster** [comment]
> @digi-booster remove the UTM content please

done!

**Review by mihajlija — APPROVED** [review]

**Pull request merged**